.github/workflows/example-basic-pnpm.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Cypress v9 and lower ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ #
```

Comment:
I suppose "and lower" doesn't accurately describe this. I think we can just edit the comment to read "Cypress v9" in all applicable areas to avoid any confusion. I don't mind "v10 and higher" since it conveys the larger change in post v9 and expands to the latest Cypress version.

Reply:
@jaffrepaul 

Instead of changing to "Cypress v9" (knocking off "and lower"), I suggest instead to align with the documentation for [Cypress legacy configuration](https://docs.cypress.io/guides/references/legacy-configuration) which says "This guide is for Cypress 9 and below" and change the header title instead to

"Cypress v9 and below (using Legacy configuration)"

which reflects the fact that the v9 tests are representing all legacy configurations.

It's your call. I will change the title to what you think is best, either:
- "Cypress v9" or
- "Cypress v9 and below (using Legacy configuration)"

Let me know.